src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -87,7 +98,28 @@ It supports the following flags:
 
 ### AWS specific
 
-- `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html). Only versions *11.3.1+ support this feature.
+- `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html).
+
+### OpenStack specific
+
+- `--cloud` - Name of the cloud in the `cloud-config` secret. This is almost always "openstack".
+- `--cloud-config` - Name of the `cloud-config` secret which defines the credentials for the OpenStack project in which the cluster should be created. This must be created in the organization namespace before creating a cluster.
+- `--external-network-id` - UUID of the external network to be used. Only required if multiple external networks are available.
+- `--failure-domain` - Failure domain of worker nodes.
+- `--node-cidr` - CIDR defining the IP range of cluster nodes.
+- `--node-image-uuid` - UUID of the root volume image source for the control plane and worker machines.
+- `--bastion-machine-flavor` - Flavor (a.k.a. size) of the worker node machines.
+- `--bastion-disk-size` - Size of root volumes attached to the cluster bastion machine in gigabytes. Must be greater than or equal to the size of the bastion source image (`--bastion-image-uuid`).
+- `--bastion-image-uuid` - UUID of the root volume image source for the bastion machine.
```

Comment:
I'd like to improve those flag names before releasing this to public. We already have a conversation here https://gigantic.slack.com/archives/C01BYMF6RN0/p1644522672656689

IMO this should be `--bastion-root-volume-source-uuid`. Or we can make it `--bastion-image` and add extra `--boot-from-volume` boolean flag (this is what `openstack` cli is doing).

The same applies to the node image.

Reply:
It would have been better to have had this feedback on https://github.com/giantswarm/kubectl-gs/pull/651. I will make a new PR to `kubectl-gs` and update these docs once released.

Reply:
Sorry I missed that there.

Reply:
Done in https://github.com/giantswarm/kubectl-gs/pull/678

Reply:
I will update this PR once `kubectl-gs` and `cluster-openstack` are released.